### PR TITLE
Remove secondary master

### PR DESCRIPTION
**Pull request by apc999**
### What changes are proposed in this pull request?

Remove secondary master

### Why are the changes needed?

Secondary Master is no longer used

### Does this PR introduce any user facing changes?

- Property key `alluxio.secondary.master.metastore.dir` is removed

### CODE REVIEW

File: bin/alluxio-start.sh [hunk 1]
```diff
@@ -172,19 +169,6 @@ start_master() {
...
-  if [[ ${ALLUXIO_MASTER_SECONDARY} == "true" ]]; then
-    if [[ `${LAUNCHER} ${BIN}/alluxio getConf ${ALLUXIO_MASTER_JAVA_OPTS} alluxio.master.journal.type` == "EMBEDDED" ]]; then
-      echo "Secondary master is not supported for journal type: EMBEDDED"
-      exit 1
-    fi
-    echo "Starting secondary master @ $(hostname -f). Logging to ${ALLUXIO_LOGS_DIR}"
-    (nohup ${BIN}/launch-process secondary_master > ${ALLUXIO_LOGS_DIR}/secondary_master.out 2>&1) &
-  else
-    echo "Starting master @ $(hostname -f). Logging to ${ALLUXIO_LOGS_DIR}"
-    (JOURNAL_BACKUP="${journal_backup}" nohup ${BIN}/launch-process master > ${ALLUXIO_LOGS_DIR}/master.out 2>&1) &
```
Comment (Xenorith) [inline bin/alluxio-start.sh:184]: keep the else branch... need to start normal master

**Review by Xenorith — COMMENTED** [review]
`2023-07-22T03:38:17.6181683Z [ERROR] /usr/src/alluxio/dora/minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java:33:1: Extra separation in import group before 'javax.annotation.concurrent.NotThreadSafe' [CustomImportOrder]`

File: bin/alluxio-start.sh (hunk 1 above)
Comment (apc999) [reply bin/alluxio-start.sh:184]: oops

**Comment by apc999** [comment]
@Xenorith  PTAL

**Review by Xenorith — APPROVED** [review]

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks!

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**